File: LatPrak2_2 (C++)/Product.cpp
class Product{
	private:
	//atribut dalam class Product
		string price;
		string id_product;

	public:
		//constructor
		Product(){
		}

		//prosedur dan fungsi untuk get & set dari atribut Price
		void setPrice(string price){
			this->price = price;
		}
		
		string getPrice(){
			return this->price;
		}

		//prosedur dan fungsi untuk get & set dari atribut id product
		void setIdProduct(string id_product){
			this->id_product = id_product;
		}
		
		string getIdProduct(){
			return this->id_product;
		}
	//destructor
	~Product(){
	}
};

File: LatPrak2_2 (PHP)/Product.php
<?php 
 
 class Product{

	//atribut dalam class product
	private $price = "";
	private $id_product = "";

		
	//constructor
	public function __construct($price = "", $id_product = ""){
		$this->price = "";
		$this->id_product = "";
	}

	//set and get for atribut Price
	public function setPrice($price){
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	//set and get for atribut Id product
	public function setIdProduct($id_product){
		$this->id_product = $id_product;
	}

	public function getIdProduct(){
		return $this->id_product;
	}

	function __destruct(){
	}

}

?>